client/.gitignore

```diff
@@ -1,4 +1,6 @@
 .DS_Store
 node_modules/
 out/
-npm-debug.log
\ No newline at end of file
+npm-debug.log
+languageServerExample.log
+server/
```

Comment:
Please remove the last line (as the server extension is obsolete)

Reply:
This is not "server" as in the directory that holds the code for the legacy typescript server. That's the folder that held javascript files that bloated this PR.